CHANGELOG.md

```diff
@@ -10,10 +10,16 @@ Current
 
 ### Added:
 
+- [Support for Druid's In Filter](https://github.com/yahoo/fili/pull/64)
```

Comment:
Should add a note that it only works with v0.9.1 (I think... I don't think it was there in earlier versions)